Makefile

```diff
@@ -91,36 +91,42 @@ endif
 debug-console:
 	bash -f ./functional/debug_console/run.sh
 
+EXTRA_GINKGO_FLAGS :=
+GINKGO_FLAGS := -failFast -v -focus "${FOCUS}" -skip "${SKIP}" $(GINKGO_TEST_FLAGS)
+
+GINKGO_TEST_FLAGS := ./integration/docker/ -- -runtime=${RUNTIME} -timeout=${TIMEOUT} -hypervisor=$(KATA_HYPERVISOR)
+
+ifeq ($(KATA_HYPERVISOR),firecracker)
+GINKGO_FLAGS=-skip "${SKIP_FIRECRACKER}"
+endif
+
+ifeq ($(KATA_HYPERVISOR),cloud-hypervisor)
+GINKGO_FLAGS=-skip "${SKIP_CLH}"
+endif
+
+RUN_PARALLEL :=
```

Comment:
I can see that the code will work, but I'd still rather we default this to `true` as currently we're saying "if the variable is unset, it's true which is slightly counter-intuitive.